js/apps/system/_admin/aardvark/APP/frontend/js/templates/indicesView.ejs

```diff
@@ -485,25 +492,80 @@
                   <th>
                     <input id="newTtlBackground" type="checkbox" name="newTtlBackground" value="true">
                   </th>
-                  <th class="tooltipInfoTh">
-                    <div class="tooltipDiv">
-                      <a class="index-tooltip" data-toggle="tooltip" data-placement="left" title="Create the index in background.">
-                        <span rel="tooltip" class="arangoicon icon_arangodb_info"></span>
-                      </a>
-                    </div>
-                  </th>
+                    <th class="tooltipInfoTh">
+                        <div class="tooltipDiv">
+                            <a class="index-tooltip" data-toggle="tooltip" data-placement="left"
+                               title="Create the index in background.">
+                                <span rel="tooltip" class="arangoicon icon_arangodb_info"></span>
+                            </a>
+                        </div>
+                    </th>
                 </tr>
               </table>
             </div>
+            <div id="newIndexTypeZkd" class="newIndexClass" style="display:none">
+                <table>
+                    <tr>
+                        <th class="collectionTh">Fields:</th>
+                        <th><input type="text" id="newZkdFields" value=""/></th>
+                        <th class="tooltipInfoTh">
+                            <div class="tooltipDiv">
+                                <a class="index-tooltip" data-toggle="tooltip" data-placement="left"
+                                   title="A comma-separated list of attribute paths.">
+                                    <span rel="tooltip" class="arangoicon icon_arangodb_info"></span>
+                                </a>
+                            </div>
+                        </th>
+                    </tr>
+                    <tr>
+                        <th class="collectionTh">Name:</th>
+                        <th><input type="text" id="newZkdName" value=""/></th>
+                        <th class="tooltipInfoTh">
+                            <div class="tooltipDiv">
+                                <a class="index-tooltip" data-toggle="tooltip" data-placement="left"
+                                   title="Index name. If left blank, a name will be auto-generated. Example: byValue">
+                                    <span rel="tooltip" class="arangoicon icon_arangodb_info"></span>
+                                </a>
+                            </div>
+                        </th>
+                    </tr>
+                    <tr>
+                        <th class="collectionTh">Field Value Types:</th>
+                        <th><input type="text" id="newZkdFieldValueTypes" value="double" disabled/></th>
```

Comment:
1.) Would be nice if the input field also is styled (css) as a disabled input field. From first look it seems the user is allowed to edit it.
2.) Additional candy: When hovering over a disabled input field - you could change the cursor as well to indicate even further, css: `cursor: not-allowed`

Reply:
IMHO 1 should be done, 2 optional :P

Reply:
Fixed.